pkg/thing/delivery/amqp/client.go

```diff
@@ -50,6 +52,20 @@ func (mp *msgClientPublisher) SendRegisterDevice(msg network.RegisterResponseMsg
 	return mp.amqp.PublishPersistentMessage(exchangeFogOut, registerOutKey, jsonMsg)
 }
 
+// SendUnregisterDevice sends an unregistered message
+func (mp *msgClientPublisher) SendUnregisterDevice(thingID string, errMsg *string) error {
+	mp.logger.Debug("Sending unregister message")
+
```

Comment:
I think it's better to remove this blank line.

Reply:
Done!